scripts/just.config.ts

```diff
@@ -16,7 +16,6 @@ const { webpack, webpackDevServer } = require('./tasks/webpack');
 const { verifyApiExtractor, updateApiExtractor } = require('./tasks/api-extractor');
 const lintImports = require('./tasks/lint-imports');
 const prettier = require('./tasks/prettier');
-const bundleSizeCollect = require('./tasks/bundle-size-collect');
```

Comment:
Are we removing bundle size collection completely here? I see we're moving the file under `scripts` but I don't see any replacement for removing these here.

Reply:
I just changed how it's called to be the same as in master. The motivation was that the bundle size collect just task mysteriously started hanging my PR builds?? So since I noticed master was calling the task differently, I tried that here and it worked.